src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -349,6 +351,13 @@ public async Task AddNode()
 
                 nodeInfo = infoCmd.Result;
 
+                if (SchemaUpgrader.CurrentVersion.ServerVersion != nodeInfo.ServerSchemaVersion)
```

Comment:
What if we have a 4.2 cluster with schema 15 and we want to add a new node with schema 16 (also 4.2). This should be supported, right? We cannot only add node with 'older' schema?

Reply:
No it shouldn't... you don't know what changes there were.
for example changing tree to table.. like we did with identities.